src/Control/Middleware/URLSpecialsMiddleware.php

```diff
@@ -17,8 +17,6 @@
  *
  * The rules are:
  *  - flush GET parameter
- *  - isDev GET parameter
- *  - isTest GET parameter
```

Comment:
Not strictly related but these params were removed a while ago so removing them from the PHPDoc while I'm poking around